docs/usage.rst

```diff
@@ -5,13 +5,12 @@ Usage
 Configuration file location
 ---------------------------
 
-Before running the linter, you need to supply configuration in a file, in INI format. Import Linter will look in the
-current directory for one of the following files:
-
+Before running the linter, you need to supply configuration in a file.
+If not specified over the command line, import linter will check the following files in that order:
```

Comment:
We've lost the 'look in the current directory'...would be good to add that back in.

Also would it be good to clarify that it expects different configuration format depending on the file?

Reply:
done